extensions/amp-brid-player/validator-amp-brid-player.protoascii

```diff
@@ -26,12 +26,8 @@ tags: {  # amp-brid-player
   }
   attr_lists: "common-extension-attrs"
 }
-tags: {  # <amp-brid-player>
-  html_format: AMP
-  tag_name: "AMP-BRID-PLAYER"
-  disallowed_ancestor: "AMP-SIDEBAR"
-  requires_extension: "amp-brid-player"
-  attrs: { name: "autoplay" }
+attr_lists: { # <amp-brid-player> attributes that are common to both AMP & A4A.
```

Comment:
I am not sure if we want to allow `fullscreen-on-landscape` in A4A, @ampproject/a4a  Do we want to allow A4A Ads to take a video ad to fullscreen on device rotation? I don't have a strong opinion against it, just erring on the side of caution.